sdk/src/main/scala/com/horizen/account/forger/AccountForgeMessageBuilder.scala

```diff
@@ -315,7 +315,7 @@ class AccountForgeMessageBuilder(
     header.bytes.length
   }
 
-  override def collectTransactionsFromMemPool(nodeView: View, blockSizeIn: Int, mainchainBlockReferenceDataToRetrieve: Seq[MainchainHeaderHash], timestamp: Long, forcedTx: Iterable[SidechainTypes#SCAT]) : Seq[SidechainTypes#SCAT] = {
+  override def collectTransactionsFromMemPool(nodeView: View, blockSizeIn: Int, mainchainBlockReferenceDataToRetrieve: Seq[MainchainBlockReferenceData], timestamp: Timestamp, forcedTx: Iterable[SidechainTypes#SCAT]): Seq[SidechainTypes#SCAT] = {
```

Comment:
let;s change the name as well: `mainchainBlockReferenceDataToRetrieve` -> `mainchainBlockReferenceData`